src/main/java/uk/gov/justice/digital/delius/data/api/InstitutionalReport.java

```diff
@@ -17,6 +18,7 @@
     private Long offenderId;
     private Conviction conviction;
 
+    @JsonIgnore
```

Comment:
Are you sure? Won't this break clients dependent on this deprecation?

Reply:
I put that in whilst debugging some issues with ObjectMapper - good catch, it's gone now